examples/dcmip_3-1_linear_advection_triangle_bdfm1.py

```diff
@@ -40,12 +40,14 @@
 timestepping = TimesteppingParameters(dt=10.0)
 output = OutputParameters(Verbose=True, dumpfreq=1, dirname='dcmip')
 parameters = CompressibleParameters(k=k, Omega=Omega)
+diagnostics = Diagnostics(fieldlist)
```

Comment:
I don't think so... the register method is only called if steady_state_dump_err is True and in that case it registers the error field (which is f - f_init). At least, that's what I want to happen!


________________________________
From: Lawrence Mitchell <notifications@github.com>
Sent: 28 April 2016 11:47
To: firedrakeproject/dcore
Cc: Shipton, Jemma; Author
Subject: Re: [firedrakeproject/dcore] Linear sw (#32)


In examples/dcmip_3-1_linear_advection_triangle_bdfm1.py<https://github.com/firedrakeproject/dcore/pull/32#discussion_r61407723>:

> @@ -40,12 +40,14 @@
>  timestepping = TimesteppingParameters(dt=10.0)
>  output = OutputParameters(Verbose=True, dumpfreq=1, dirname='dcmip')
>  parameters = CompressibleParameters(k=k, Omega=Omega)
> +diagnostics = Diagnostics(fieldlist)


I think that these fields will be registered twice (once in initialisation here) and then again in State.dump?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/firedrakeproject/dcore/pull/32/files/380f79f1f400620c55255ebfa19a83a0fe62b304#r61407723>